Services/source/housekeeping/housekeeping_service.c

```diff
@@ -437,19 +435,14 @@ int num_digits(int num) {
   return count;
 }
 
-static inline Result prv_get_lock(SemaphoreHandle_t *lock) {
+static inline void prv_get_lock(SemaphoreHandle_t *lock) {
   if (*lock == NULL) {
     *lock = xSemaphoreCreateMutex();
     if (*lock == NULL) {
       return FAILURE;
```

Comment:
return from void function

Reply:
You could try using xSemaphoreCreateMutexStatic to guarantee it won't fail

Reply:
woops lol